### PR TITLE
Update react science dependency

### LINKED ISSUE #75: Update react-science dependency
There are couple of breaking changes ...

### PR DESCRIPTION
**Pull request by moonayyur**
closes #75

### CODE REVIEW

File: src/components/layout/Header.tsx [hunk 1]
```diff
@@ -40,43 +40,40 @@ function PixeliumHeader() {
...
-          titleOrientation="vertical"
+          icon={<FaBug />}
           onClick={() => {
             openLogs();
             markAsRead();
           }}
         >
-          <FaBug />
-          {unreadCount > 0 && (
+          {/*TODO: find a way to display the unread count chip on the toolbar*/}
```
Comment (moonayyur) [inline src/components/layout/Header.tsx:65]: @stropitek should we open an issue in react-science to add the chip feature in the toolbar component ?

Comment (stropitek) [reply src/components/layout/Header.tsx:65]: I just made a release, it includes a new `tag` prop on buttons and toolbar items that does exactly that

File: src/components/layout/CenterPanel.tsx [hunk 2]
```diff
@@ -57,16 +60,34 @@ function CenterPanel() {
...
+            selectedTabId={currentTab}
+            onChange={openTab}
+            css={css`
+              height: 100%;
+              div[role='tablist'] {
+                overflow-x: auto;
+                overflow-y: hidden;
+              }
+              div[role='tabpanel'] {
+                height: calc(100% - 30px);
```
[the remark this answers is not in the document]
Comment (stropitek) [reply src/components/layout/CenterPanel.tsx:77]: It's quiet an old issue. I suppose that if the Tab2 component does not exist, it must have either been removed or the `Tab` component was removed and `Tab2` was renamed to `Tab`

**Comment by stropitek** [comment]
https://75-update-react-science-depe.pixelium.pages.dev/

File: src/components/layout/CenterPanel.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (stropitek) [reply src/components/layout/CenterPanel.tsx:77]: It seems possible to have a scroll bar without this hack:
https://react-science.pages.dev/stories/?path=/story/components-tabs--many-tabs

Can you try to have a similar implementation?

[the remark this answers is not in the document]
Comment (moonayyur) [reply src/components/layout/CenterPanel.tsx:77]: The scrollbar implementation for tablist (the titles) is the same as in react-science 

```
css={css`
        height: 100%;
        div[role='tablist'] {
          overflow-x: ${orientation === 'vertical' ? 'hidden' : 'auto'};
          overflow-y: ${orientation === 'vertical' ? 'auto' : 'hidden'};
        }
```

**Review by stropitek — APPROVED** [review]

**Pull request merged**